prereise/gather/griddata/hifld/calculate/remap.py

```diff
@@ -0,0 +1,60 @@
+import copy
+
+
+def get_zone_mapping(zone):
+    """Generate dictionaries of zone using the zone.csv
+
+    :param pandas.DataFrame zone: the zone.csv data
+    :return: (*tuple*) -- a tuple of two dictionaries, where the first
+    dictionary is a dict mapping the STATE to its ID and the second is a dict
+    mapping a tuple of state and interconnect to its ID.
+    """
+
+    zone_dic = {}
+    zone_interconnect_dic = {}
+    for i in range(len(zone)):
+        tu = (zone["zone_name"][i], zone["interconnect"][i])
+        zone_dic[zone["zone_name"][i]] = zone["zone_id"][i]
+        zone_interconnect_dic[tu] = zone["zone_id"][i]
```

Comment:
Is this equivalent to:
```python
zone_dic = pd.Series(zone.zone_id, index=zone.zone_name"].to_dict()
zone_interconnect_dic = pd.Series(zone.zone_id, index=zip(zone.zone_name, zone.interconnect)).to_dict()
```
?
We could also skip the `to_dict` call and just return Series objects.

Reply:
Thanks for the pandas recommendation!

As far as the `to_dict` called, this data gets accessed in dictionary form later on in the (`data_trans.py`) code. The conversion can either happen here or wherever it gets accessed, but the `zone_dic` dictionary is already used in the pandas query in `clean.py` and both will be used to write the data to the csv's expected by our existing `Grid` object. I would say it's easier to refactor everything after the fact to just Series objects than do that now.

Reply:
Why do we need this function at all?

Reply:
By the way the `zone_dic` won't work to query the substations in your `clean` module since `STATE` in the substation file are state abbreviation and the key of `zone_dic` are load zone name not abbreviated.